augur/titer_model.py

```diff
@@ -55,27 +55,33 @@ def load_from_file(filename, excluded_sources=None):
         measurements = defaultdict(list)
         strains = set()
         sources = set()
+        titer_files = [filenames] if type(filenames)==str else filenames
 
-        with myopen(filename, 'r') as infile:
-            for line in infile:
-                entries = line.strip().split('\t')
-                try:
-                    val = float(entries[4])
-                except:
-                    continue
-                test, ref_virus, serum, src_id = (entries[0], entries[1],entries[2],
-                                                  entries[3])
+        for fname in titer_files:
+            if not os.path.isfile(fname):
+                print("ERROR: titer file %s does not exist!"%fname)
```

Comment:
Instead of printing an error to stdout, we should let missing files raise an exception during the `open` call and stop execution. We can enforce the existence of files or handle the exception cleanly at the command line interface.

I will push a new commit for this.

Reply:
makes sense!